opentreemap/treemap/js/src/lib/utility.js

```diff
@@ -143,7 +143,7 @@ exports.makeZoomLatLngQuery = function(zoomLatLng) {
 exports.getPolygonDisplayArea = function(poly) {
     function totalAreaInMeters(collection) {
         if (_.isArray(collection[0])) {
-            return _.chain(collection)
+            return _(collection)
                     .map(totalAreaInMeters)
                     .reduce(function(sum, num) {
                         return sum + num;
```

Comment:
`reduce` here returns a scalar floating point number, which has no `value` method. So the call to `value` on line 151 gets a `TypeError`.

Reproduce by navigating to Plan, adding a scenario, adding one or more trees, and clicking to draw the distribution area. Click to make some points. Try to close the polygon. `TypeError`.